astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -197,24 +197,13 @@ def test_future_altaz():
     location = EarthLocation(lat=0 * u.deg, lon=0 * u.deg)
     t = Time("J2161")
 
-    # check that these message(s) appear among any other warnings.  If tests are run with
-    # --remote-data then the IERS table will be an instance of IERS_Auto which is
-    # assured of being "fresh".  In this case getting times outside the range of the
-    # table does not raise an exception.  Only if using IERS_B (which happens without
-    # --remote-data, i.e. for all CI testing) do we expect another warning.
+    # check that these message(s) appear among any other warnings
     if PYTEST_LT_8_0:
-        ctx1 = ctx2 = nullcontext()
+        ctx1 = nullcontext()
     else:
         ctx1 = pytest.warns(erfa.core.ErfaWarning)
-        ctx2 = pytest.warns(AstropyWarning, match=".*times are outside of range.*")
-    with ctx1, ctx2, pytest.warns(
+    with ctx1, pytest.warns(
         AstropyWarning,
         match="Tried to get polar motions for times after IERS data is valid.*",
     ) as found_warnings:
```

Comment:
I guess with the rest of the change we can remove `as found_warnings`

Reply:
Ah, yup